asyncio_mqtt/client.py

```diff
@@ -218,6 +227,18 @@ async def force_disconnect(self) -> None:
         if not self._disconnected.done():
             self._disconnected.set_result(None)
 
+    def _outgoing_call(self, func: Callable) -> Callable[..., Awaitable]:
+        async def decorated(*args: Any, timeout: int = 10, **kwargs: Any) -> None:
+            if not self._outgoing_calls_sem:
+                func(*args, timeout=timeout, **kwargs)
```

Comment:
Just `return await func(...)` to preserve the returned value. :+1:  Also note the missing `await`.